assets/js/index_translate.js

```diff
@@ -0,0 +1,75 @@
+// The fetchLang function is fetching the the language from the lang.json file
+// This file is just for the index page
+
+let fetchLang = async () => {
+  let response = await fetch("assets/js/lang.json");
+  let data = await response.json();
+
+  let ar = document.querySelector("#ar");
+  let en = document.querySelector("#en");
+  let ar_style = document.getElementById("ar_style");
+
+  ar.addEventListener("click", () => {
+    translate_btn.innerHTML = "Arabic";
+
+    publications.innerHTML = data.ar.publications;
+    talks.innerHTML = data.ar.talks;
+    hero_text.innerHTML = data.ar.hero_text;
+    mission_text.innerHTML = data.ar.mission_text;
+    mission.innerHTML = data.ar.mission;
+    highlighted_projects.innerHTML = data.ar.highlighted_projects;
+    masader.innerHTML = data.ar.masader;
```

Comment:
when you override the innerHTML content you are removing some stuff like links to the code, paper, and web. For example the masader `card-title` has the following 
```html
<h5 class="card-title" id="masader">Masader 
<a href="https://t.co/rGIpZGyWlJ" target="_blank" class="btn btn-color btn-sm right mr-2"><i class="fa fa-file"></i> Paper</a>
<a href="https://github.com/ARBML/masader" target="_blank" class="btn btn-color btn-sm right mr-2"><i class="fa fa-github"></i> Code</a>
<a href="https://arbml.github.io/masader/" target="_blank" class="btn btn-
</h5>
```

Reply:
Same thing for News. It contains some hyperlinks which are overridden when you do the translation.